tests/infer/test_inference.py

```diff
@@ -243,7 +256,13 @@ def guide(alpha0, beta0, alpha_n, beta_n):
         pyro.sample("lambda_latent", gamma_dist(alpha_q, beta_q))
 
     adam = optim.Adam({"lr": .0003, "betas": (0.97, 0.999)})
-    elbo = elbo_impl(strict_enumeration_warning=False)
+    if elbo_impl is RenyiELBO:
+        elbo = elbo_impl(alpha=0.2, num_particles=3, strict_enumeration_warning=False)
+        if gamma_dist is ShapeAugmentedGamma:
```

Comment:
I would instead use `with xfail_if_not_implemented():` here and raise `NotImolementedError` inside `Reny_ELBO` in multiple score_parts terms are nonzero.

Reply:
I'll do it.